src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -66,10 +66,15 @@ private void assignStalledComponents(Command command, String commandText) {
         stalledCommandText = commandText;
     }
 
+    private boolean isYes(String commandText) {
+        return commandText.equalsIgnoreCase("yes") || commandText.equalsIgnoreCase("ye")
+            || commandText.equalsIgnoreCase("y");
+    }
+
```

Comment:
I'm feeling like this is starting to go into parsing territory hmmm... Should we put this in `GradPadParser` instead? Then below we can just call `gradPradParser.isYesCommandWord` or something

Reply:
okok i'll work on it!